src/azure-cli/azure/cli/command_modules/rdbms/_transformers.py

```diff
@@ -33,3 +33,21 @@ def table_transform_output_list_servers(result):
         new_entry['Availability zone'] = key['availabilityZone']
         table_result.append(new_entry)
     return table_result
+
+
+def table_transform_output_list_sku(result):
+    table_result = []
+    skus_tiers = result[0]["supportedFlexibleServerEditions"]
```

Comment:
If you do this, you should ideally check if result has any entry. Otherwise, an empty result will throw Index out of bounds Exception

Reply:
Will do. thanks!